docs/src/Popover.doc.js

```diff
@@ -1,261 +1,1316 @@
-              <Text align="center">
-                Popover with a caret, not a 🥕
-              </Text>
+            <Box width={360}>
+              <Box flex="grow" marginEnd={4} marginStart={4}  marginTop={6} marginBottom={8}>
+                <Flex direction="column" gap={6}>
+                  <Text align="center" color="darkGray" weight="bold">
+                    Save to board
+                  </Text>
+                  <SearchField
+                    accessibilityLabel="Demo Search boards field"
+                    id="searchField"
+                    onChange={() => {}}
+                    placeholder="Search boards"
+                    size="lg"
+                  />
+                </Flex>
+              </Box>
+              <Box height={300} overflow="scrollY">
+                <Box marginEnd={4} marginStart={4}>
+                  <Flex direction="column" gap={8}>
+                    <List title="Top choices"/>
+                    <List title="All boards"/>
+                  </Flex>
+                </Box>
+              </Box>
             </Box>
           </Popover>
-        </Layer>}
-    </Box>
+        </Layer>
+      )}
+    </React.Fragment>
   );
 }
-`}
+  `}
+      />
+    </MainSection.Subsection>
+  </MainSection>,
+);
+
+card(
+  <MainSection
+    name="Localization"
+    description="Be sure to localize any text elements within Popover. Note that localization can lengthen text by 20 to 30 percent."
   />,
 );
 
 card(
-  <Example
-    id="errorPopover"
-    name="Example: ErrorPopover"
-    description={`Popover can also take on additional roles. Like [TextField](#TextField) and [TextArea](#TextArea), this component
-can be used to highlight errors on other types of form fields by setting the \`color\` to \`red.\``}
-    defaultCode={`
-function ErrorPopoverExample() {
+  <MainSection name="Variants">
+    <MainSection.Subsection
+      title="Size"
+      description={`
+The maximum width of Popover. Popover has different size configurations:
+- \`"xs"\`: 180px
+- \`"sm"\`: 230px
+- \`"md"\`: 284px
+- \`"lg"\`: 320px
+- \`"xl"\`: 360px
+- \`number\`:  Use this prop to create custom-size Popovers in pixels.
+- \`flexible\`: Use this configuration for larger Popovers. Without a defined maximum width,  Popover grows to fit the content in \`children\`.
+
+We recommend using \`"xs\`" for education Popovers and \`"xl\`" for more complex Popovers. Avoid using other configurations whenever possible as they are legacy sizes.
+      `}
+    />
+
+    <MainSection.Subsection
+      title="Color and caret"
+      description={`
+When building in-product education, be sure to pass in \`color="blue"\` and \`showCaret="true"\`, as seen in left example, and use [Experience HQ](https://ehq.pinadmin.com/) for the configuration. For Popovers that aren’t education, use the default \`color="white"\` and \`showCaret="false"\` as seen in the right example. Avoid using any other configurations as they are legacy colors.
```

Comment:
"as seen in the first example", and "as seen in the second example" (instead of left/right)

Reply:
DONE